daisy_integration_tests/daisy_e2e.test.gotmpl

```diff
@@ -137,6 +137,9 @@
     "Image import and translate ova": {
       "Path": "./image_import_and_translate_ova.wf.json"
     },
+    "Image import and translate ova with spaces in VMDK": {
```

Comment:
...spaces in VMDK file name"

It's VMDK files that have space, not the file itself. File names and other references in this PR should be updated as well.